trunner/device.py

```diff
@@ -39,12 +40,44 @@ def wait_for_dev(port, timeout=0):
 
     # naive wait for dev
     while not os.path.exists(port):
-        time.sleep(0.5)
-        asleep += 0.5
+        time.sleep(0.01)
+        asleep += 0.01
         if timeout and asleep >= timeout:
             raise TimeoutError
 
 
+def switch_rpi_ports(state):
```

Comment:
maybe `power_usb_ports(enabled: bool)`? 

than we can skip string parsing and write 
```python
'-a', f'{1 if enable else 0}'`
```

also calling the function is readable `power_usb_ports(False)`